src/data/thermal-conductivity-constants.ts

```diff
@@ -16,6 +16,369 @@ export const DIPPRThermalConductivityDictionary: Record<
+  },
+  Benzene:{
+    name:'Benzene',
+    formula:'C6H6',
+    CAS:'71-43-2',
+    molecularWeight:78.11184,
+    C1:0.23444,
+    C2:-0.00030572,
+    C3:NaN,
+    C4:NaN,
+    C5:NaN,
+    minimumTemperature:278.68,
+    maximumTemperature:413.1,
+    thermalConductivityAtMinimumTemperature:0.1492,
+    thermalConductivityAtMaximumTemperature:0.1081
+  },
+  Benzenethiol:{
+    name:'Benzenethiol',
+    formula:'C6H6S',
+    CAS:'108-98-5',
+    molecularWeight:110.17684,
+    C1:0.20996,
+    C2:-0.0002146,
+    C3:NaN,
+    C4:NaN,
+    C5:NaN,
+    minimumTemperature:258.27,
+    maximumTemperature:442.29,
+    thermalConductivityAtMinimumTemperature:0.1545,
+    thermalConductivityAtMaximumTemperature:0.1150
+  },
+  BenzoicAcid:{
+    name:'Benzoic acid',
+    formula:'C7H6O2',
+    CAS:'65-85-0',
+    molecularWeight:122.12134,
+    C1:0.2391,
+    C2:-0.0002325,
+    C3:NaN,
+    C4:NaN,
+    C5:NaN,
+    minimumTemperature:395.45,
+    maximumTemperature:596,
+    thermalConductivityAtMinimumTemperature:0.1472,
+    thermalConductivityAtMaximumTemperature:0.1005
+  },
+  Benzonitrile:{
+    name:'Benzonitrile',
+    formula:'C7H5N',
+    CAS:'100-47-0',
+    molecularWeight:103.1213,
+    C1:0.20603,
+    C2:-0.00021023,
+    C3:NaN,
+    C4:NaN,
+    C5:NaN,
+    minimumTemperature:260.28,
+    maximumTemperature:464.15,
+    thermalConductivityAtMinimumTemperature: 0.1513,
+    thermalConductivityAtMaximumTemperature:0.1085
+  },
+  Benzophenone:{
+    name:'Benzophenone',
+    formula:'C13H10O',
+    CAS:'119-61-9',
+    molecularWeight:182.2179,
+    C1:0.25867,
+    C2:-0.00022516,
+    C3:NaN,
+    C4:NaN,
+    C5:NaN,
+    minimumTemperature:321.35,
+    maximumTemperature:664,
+    thermalConductivityAtMinimumTemperature:0.1863,
+    thermalConductivityAtMaximumTemperature:0.1092
+  },
+  BenzylAlcohol:{
+    name:'Benzyl alcohol',
+    formula:'C7H8O',
+    CAS:'100-51-6',
+    molecularWeight:108.13782,
+    C1:0.17847,
+    C2:-0.000065843,
+    C3:NaN,
+    C4:NaN,
+    C5:NaN,
+    minimumTemperature:257.85,
+    maximumTemperature:478.6,
+    thermalConductivityAtMinimumTemperature:0.1615,
+    thermalConductivityAtMaximumTemperature:0.1470
+  },
+  BenzylEthylether:{
+    name:'Benzyl ethyl ether',
+    formula:'C9H12O',
+    CAS:'539-30-0',
+    molecularWeight:136.19098,
+    C1:0.2029,
+    C2:-0.0002226,
+    C3:NaN,
+    C4:NaN,
+    C5:NaN,
+    minimumTemperature:275.65,
+    maximumTemperature:528.6,
+    thermalConductivityAtMinimumTemperature:0.1415,
+    thermalConductivityAtMaximumTemperature:0.0852
+  },
+  BenzylMercaptan:{
+    name:'Benzyl mercaptan',
+    formula:'C7H8S',
+    CAS:'100-53-8',
+    molecularWeight:124.20342,
+    C1:0.20316,
+    C2:-0.00019912,
+    C3:NaN,
+    C4:NaN,
+    C5:NaN,
+    minimumTemperature:243.95,
+    maximumTemperature:472.03,
+    thermalConductivityAtMinimumTemperature:0.1546,
+    thermalConductivityAtMaximumTemperature:0.1092
+  },
```

Comment:
Add spaces after the compound data.
```suggestion
  },
  
```